src/test/java/seedu/placebook/logic/commands/FindTagsCommandTest.java

```diff
@@ -67,6 +67,26 @@ public void execute_zeroKeywords_noPersonFound() {
         assertEquals(Collections.emptyList(), model.getFilteredPersonList());
     }
 
+    @Test
+    public void execute_oneKeyword_noPersonFound() {
+        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
+        PersonHasTagsPredicate predicate = preparePredicate("enemies");
+        FindTagsCommand command = new FindTagsCommand(predicate);
+        expectedModel.updateFilteredPersonList(predicate);
+        assertCommandSuccess(command, model, uiStub, expectedMessage, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredPersonList());
+    }
+
+    @Test
+    public void execute_oneKeyword_onePersonFound() {
```

Comment:
I think can add few test case:  
one keyword and multiplePersonFound
multipleKeywords and onePerson

Reply:
alrighty